universal-editor-product-picker/src/universal-editor-ui-1/web-src/src/catalog-service/queries/categories.graphql.js

```diff
@@ -0,0 +1,17 @@
+const query = `query getCategoriesInCategory($id: String!) {
```

Comment:
It will be nice to replace on a `query-builder` that allows to avoid some duplication